src/igvfd/schemas/auxiliary_set.json

```diff
@@ -59,14 +59,44 @@
                 "oligo-conjugated lipids",
                 "oligo-conjugated antibodies",
                 "quantification barcode sequencing"
-            ]
+            ],
+            "submissionExample": {
+                "appscript": "cell hashing",
+                "igvf_utils": "cell hashing"
+            }
+        },
+        "construct_libraries": {
+            "permission": "import_items",
+            "comment": "Do not submit. This property has been moved to Sample objects.",
+            "title": "Construct Libraries",
+            "description": "The construct libraries of vectors introduced to the biosample used to produce data in this auxiliary set.",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "title": "Construct Library",
+                "type": "string",
+                "linkTo": "ConstructLibrary"
+            }
+        },
+        "moi": {
+            "permission": "import_items",
+            "comment": "Do not submit. This property has been moved to Sample objects.",
+            "title": "Multiplicity Of Infection",
+            "description": "The actual multiplicity of infection (MOI) for vectors introduced to the biosample used to produce data in this auxiliary set. At least one construct library must be specified in order to specify MOI. This property should capture the actual MOI, and not the targeted MOI.",
+            "type": "number",
+            "minimum": 0
```

Comment:
These properties were removed in https://igvf.atlassian.net/browse/IGVF-1177 
https://github.com/IGVF-DACC/igvfd/pull/614/files#diff-7388090153d2005e8176cfd0532c9884450fad9278294e111b69b19827126cf7, but I think you accidentally re-added them during rebasing.